src/pyj/read_book/view.pyj

```diff
@@ -107,6 +107,19 @@ def show_controls_help():
     def msg(txt):
         return set_css(E.div(txt), padding='1ex 1em', text_align='center', margin='auto')
 
+    left_msg = msg(_('Tap to turn back'))
+    left_width = '25vw'
+    right_msg = msg(_('Tap to turn page'))
+    right_width = '75vw'
+    if rtl_page_progression():
+        left_msg, right_msg = right_msg, left_msg
+        left_width, right_width = right_width, left_width
+
+    # Clear it out if this is not the first time it's created.
+    # Needed to correctly show it again in a different page progression direction.
+    if container.firstChild:
+        container.removeChild(container.firstChild)
```

Comment:
This was actually producing a doubled tutorial overlay before, since it's called twice, so the first time I tried this, the last book's tutorial overlay kept showing up, even if the new book was a different direction.